From 07d22529473492468215a8bfe07d56838f548921 Mon Sep 17 00:00:00 2001
From: OpenStack Release Bot <infra-root@openstack.org>
Date: Tue, 8 Mar 2022 12:05:23 +0000
Subject: [PATCH] Add Python3 zed unit tests

This is an automatically generated patch to ensure unit testing
is in place for all the of the tested runtimes for zed.

See also the PTI in governance [1].

[1]: https://governance.openstack.org/tc/reference/project-testing-interface.html

Change-Id: I21c553d594eaf856e6b2d74ec6174394618a3156
(cherry picked from commit d42220cdc8136d59a58a70805064834c1601b13b)
---
 .zuul.yaml        | 2 +-
 ec2api/context.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.zuul.yaml b/.zuul.yaml
index a150fe9..7dfcce2 100644
--- a/.zuul.yaml
+++ b/.zuul.yaml
@@ -2,7 +2,7 @@
     templates:
       - check-requirements
       - openstack-lower-constraints-jobs
-      - openstack-python3-yoga-jobs
+      - openstack-python3-zed-jobs
       - publish-openstack-docs-pti
     check:
       jobs:
diff --git a/ec2api/context.py b/ec2api/context.py
index 9080a1a..39c3802 100644
--- a/ec2api/context.py
+++ b/ec2api/context.py
@@ -54,7 +54,7 @@ class RequestContext(context.RequestContext):
         super(RequestContext, self).__init__(
             auth_token=auth_token,
             user=user_id or user,
-            tenant=project_id or tenant,
+            project_id=project_id or tenant,
             is_admin=is_admin,
             request_id=request_id,
             resource_uuid=kwargs.pop('resource_uuid', None),
-- 
2.35.1

